zcash_proofs/src/lib.rs

```diff
@@ -22,9 +22,9 @@ pub mod prover;
 
 pub fn load_parameters(
     spend_path: &Path,
-    spend_hash: &str,
+    // spend_hash: &str,
     output_path: &Path,
-    output_hash: &str,
+    // output_hash: &str,
     sprout_path: Option<&Path>,
     sprout_hash: Option<&str>,
```

Comment:
We can hard-code `sprout_hash` as well (as a static `&str`, not an `Option<&str>`); it is similarly fixed now.